crates/bevy_render/src/camera/camera.rs

```diff
@@ -382,8 +384,17 @@ pub fn camera_system<T: CameraProjection + Component>(
             .is_changed(&changed_window_ids, &changed_image_handles)
             || added_cameras.contains(&entity)
```

Comment:
The extra query and paramset in this system are unnecessary.
You mind cleaning it up a bit? :)
```suggestion
            || camera.is_added()
```